pgjdbc/src/main/java/org/postgresql/core/Encoding.java

```diff
@@ -122,7 +134,7 @@ public boolean hasAsciiNumbers() {
    */
   public static Encoding getJVMEncoding(String jvmEncoding) {
     if ("UTF-8".equals(jvmEncoding)) {
```

Comment:
should this ignore case, or do we know it will be all caps?

Reply:
The existing code checks for explicit equals so haven't touched any of it in this PR.

I tested that out locally and every combination of upper, lower, or mixed case, with or without a dash worked. Maybe a better approach would be to defer to the JDK to determine if the charset is UTF8 via comparing the canonical name:

```
if ("UTF-8".equals(Charset.forName(encoding).name())) {
   // We're UTF-8...
}
```

Could follow up with something for that in a separate PR. I don't think it'd break anything but it is a change as it'd expand the encoding considered UTF-8 (likely for the better).

Reply:
@sehrope, do you want to coordinate on proposed changes?

Reply:
Sure but let's keep it separate from this PR as it's a change in behavior. I'll follow up later with a PR for it.